doc/source/getting-started.rst

```diff
@@ -17,12 +17,30 @@ following things:
 * **Docker** to use containers that standardize computing environments
 * **JupyterHub** to manage users and deploy Jupyter notebooks
 
-To get started, we recommend beginning with the first item on our
-:ref:`main page <creating-your-jupyterhub>`. If you'd like to know how to
-expand and customize your jupyterhub setup, check out
-:ref:`extending-jupyterhub`. Finally, for a more extensive description of the
-tools and services that JupyterHub utilizes, see our 
-:ref:`tools` page.
+You already are well on your way to understanding what it means (procedurally)
+to deploy Jupyterhub. 
+
+For more guidance
+-----------------
+
+If you want more guidance, the next step on your journey is to :ref:`create a
+Kubernetes cluster on Google Cloud
+<google-cloud>`. 
+
+For info on extensions and customization 
```

Comment:
Maybe add an extra sentence or two describing what we mean by this. E.g.:

"If you'd like to know how to expand and customize your jupyterhub setup, such as
increasing the hardware available to users or changing authentication services, check
 out :ref:`extending-jupyterhub`."